src/main/webapp/scripts/classes/recommended-video-display-class.js

```diff
@@ -0,0 +1,49 @@
+class RecommendedVideoDisplay{
+  #CARD = { element: 'div', className: 'card'};
+  #CARDBODY = { element: 'div', className: 'card-body p-3'};
+  #THUMBNAIL = { element: 'img', className: 'card-img-top'};
+  #TITLE = { element: 'h5', className: 'card-title'};
+  #TEXT = { element: 'p', className: 'card-text'};
+
+  constructor(containerId) {
+    this.dom = document.getElementById(containerId);
+    this.dom.className = 'row mt-4';
+  }
+
+  addVideo(video) {
+    const videoCard = this.createVideoCard(video);
+    const newElement = document.createElement('div');
+    newElement.className = "col-lg-3 col-md-4 col-sm-6 col-xs-12";
+    newElement.appendChild(videoCard);
+    this.dom.appendChild(newElement);
+  }
+
+  createVideoCard(video) {
+    const card = this.newElement(this.#CARD);
+    const cardBody = this.newElement(this.#CARDBODY);
+    const thumbnail = this.newElement(this.#THUMBNAIL);
+    const title = this.newElement(this.#TITLE);
+    const text = this.newElement(this.#TEXT);
+    thumbnail.src = video.thumbnail;
+    title.innerText = video.title;
+    title.title =  video.title;
+    if(video.sentiment === null) {
+      text.innerText = video.likeCount + ' likes';
+    } else {
+      text.innerText = video.sentiment;
+    }
```

Comment:
Nit: Kind of inconsistent behavior to have one or the other. If there is no score, then is it actually a popular video? Oh or maybe I have this wrong. Is this the trending page results? Or popular videos (based on sentiment score) that are stored in the datastore?

Reply:
I did this so that the function can be reused for the positive videos page so like, if there's a sentiment (this video was from the datastore) then display it, if not (this video is from youtube API) display # of likes